assets/src/components/v2/disruption_diagram/disruption_diagram.tsx

```diff
@@ -379,7 +378,7 @@ const MiddleSlotComponent: ComponentType<MiddleSlotComponentProps> = ({
   let icon;
   if (slot.show_symbol) {
     if (isCurrentStop) {
-      if (isAffected && effect in ["station_closure", "suspension"]) {
+      if (isAffected) {
```

Comment:
Oh good thing this was changed, cuz that previous condition didn't work the way we wanted it to—`x in obj` in JS checks if `x` is in the keys of `obj`. So `x in ["station_closure", "suspension"]` is only true if `x` is 0, 1, "length", "map", etc. Fields of the array object, not their values. JS is weird!

To get the behavior we were looking for, it would have been `["station_closure", "suspension"].includes(effect)`.

Reply:
Double thumbs-up. Yeah, I think I originally was the one to add it, so I'm glad I got to fix it and realize the mistake. 👍 👍